editor.planx.uk/src/pages/FlowEditor/lib/__tests__/navigation.test.ts

```diff
@@ -15,117 +16,194 @@ beforeEach(() => {
 
 afterEach(() => setState(initialState));
 
-test("filterFlowByType() correctly returns a filtered view of the given flow", () => {
-  setState({ flow: flowWithThreeSections });
-
-  // Three Sections in test flow
-  const sectionResult = filterFlowByType(TYPES.Section);
-  expect(Object.keys(sectionResult)).toEqual(
-    expect.arrayContaining(["firstSection", "secondSection", "thirdSection"]),
-  );
-
-  // Three Statements in test flow
-  const statementResult = filterFlowByType(TYPES.Question);
-  expect(Object.keys(statementResult)).toEqual(
-    expect.arrayContaining([
-      "firstQuestion",
-      "secondQuestion",
-      "thirdQuestion",
-    ]),
-  );
-
-  // No FileUpload in test flow
-  const uploadResult = filterFlowByType(TYPES.FileUpload);
-  expect(Object.keys(uploadResult).length).toBe(0);
+describe("filterFlowByType()", () => {
```

Comment:
The tests for a flow without sections and flow with sections (not in folders) have not changed here - I've just introduced `describe()` blocks per store function to better organise things ! 

Each describe block now has a third scenario for "a flow with sections in folders"